mlir/test/Target/LLVMIR/openmp-target-private.mlir

```diff
@@ -0,0 +1,73 @@
+// RUN: mlir-translate -mlir-to-llvmir %s | FileCheck %s
```

Comment:
Please use `--split-input-file`

Reply:
There is a known issue with using `--split-input-file` on mlir code with `omp.target` in it. A discussion is [here](https://discourse.llvm.org/t/request-for-input-on-a-fix-for-a-bug-utilizing-omp-targetop-in-conjunction-with-mlir-translates-split-input-file/71785). I just opened an issue for this https://github.com/llvm/llvm-project/issues/105545